tests/test_frontend_type_conversion.py

```diff
@@ -57,6 +57,7 @@ def to_xdsl() -> Callable[..., Any]:
 
 def test_raises_exception_on_unknown_type():
     type_converter = TypeConverter(globals)
+    type_converter.file = 'file'
```

Comment:
Why is this needed?

Reply:
If the file is None it asserts False

Reply:
Can you pass it to the constructor directly then?